TDataFrame.hxx

```diff
@@ -856,182 +1070,172 @@ public:
                }
             }
 
-            BuildAllReaderValues(r, slot); // TODO: if TTreeProcessor reuses the same TTreeReader object there is no need to re-build the TTreeReaderValues
+            BuildAllReaderValues(r, slot); // TODO: if TTreeProcessor reuses the same TTreeReader object there is no
+                                           // need to re-build the TTreeReaderValues
 
             // recursive call to check filters and conditionally execute actions
-            while(r.Next())
-               for(auto& actionPtr : fBookedActions)
-                  actionPtr->Run(slot, r.GetCurrentEntry());
+            while (r.Next())
+               for (auto &actionPtr : fBookedActions) actionPtr->Run(slot, r.GetCurrentEntry());
```

Comment:
this indentation and no braces will backfire when someone adds something inside the `for` loop.
same at line 1094.